Documentation updates for clarifying what is allowed in LIMIT & Co (#588)
neo4j/docs-cypher: modules/ROOT/pages/clauses/call-subquery.adoc

to process before committing the current transaction and starting a new one.
The number of input rows is set with the modifier `OF n ROWS` (or `ROW`).
If omitted, the default batch size is `1000` rows.

> The number of rows can be expressed using any expression that evaluates to a positive integer -- however the expression cannot refer to nodes or relationships.

Reviewer: ```suggestion
The number of rows can be expressed using any expression that evaluates to a positive integer and does not refer to nodes or relationships.
```

> The following is the same example as above for loading a CSV file but with one transaction every `2` input rows:

Reviewer 2: I am not sure whether we should backquote `2` (and also`1000`) a couple lines above. But I leave it up to you or the documentation team to decide on style.

Reviewer: Both should be within backticks (`1000` because it is referring to a default value, and `2` because it references something in a code example).

> The following is the same example as above for loading a CSV file but with one transaction every `2` input rows:

Reviewer: ```suggestion
This example loads a CSV file with one transaction for every `2` input rows:
```

Author: So, you do not deem the information important, that this is only a slight modification of the previous example?

Reviewer: Not really, no. I think it works as a stand-alone example, without tempting the reader to scroll back up again.
